plugins/database_api/api.cpp

```diff
@@ -78,11 +81,7 @@ struct plugin::api_impl final {
     std::set<std::string> lookup_accounts(const std::string &lower_bound_name, uint32_t limit) const;
     uint64_t get_account_count() const;
 
-    // Witnesses
-    std::vector<optional<witness_api_object>> get_witnesses(const std::vector<witness_object::id_type> &witness_ids) const;
-    fc::optional<witness_api_object> get_witness_by_account(std::string account_name) const;
-    std::set<account_name_type> lookup_witness_accounts(const std::string &lower_bound_name, uint32_t limit) const;
-    uint64_t get_witness_count() const;
+    std::vector<optional<witness_api::witness_api_object>> get_witnesses(const std::vector<witness_object::id_type> &witness_ids) const;
```

Comment:
What reasons to keep this method?

Reply:
It is used inside database_api so i removed it from database_api plugin but left it for database needs

Reply:
Yes my bad i will remove it.